app/templates/experiments/info/results/all-loading.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="row">
+    <div class="col-xs-4 col-xs-offset-4" style="height:100%">
```

Comment:
Good catch. Strictly speaking this class can be removed entirely now. (I was tinkering with centering the loading screen on the page and decided not to; this was leftover cruft)